crates/proto/src/packets/player_action.rs

```diff
@@ -0,0 +1,33 @@
+use crate::types::network_block_pos::NetworkBlockPos;
+use bedrockrs_core::int::VAR;
+use bedrockrs_shared::actor_runtime_id::ActorRuntimeID;
+
+#[derive(Debug, Clone)]
+pub struct PlayerActionPacket {
+    pub player_runtime_id: ActorRuntimeID,
+    pub action: VAR<i32>,
```

Comment:
can you turn `action` into an enum?
all the possible values are here: https://mojang.github.io/bedrock-protocol-docs/html/enums.html
![image](https://github.com/user-attachments/assets/c481f3de-6f9a-4114-80ae-c8dd008d416b)

you can create it as simple as making a new file in the `types` directory and adding an enum like this:
```rust
#[derive(ProtoCodec, ...)]
#[enum_repr(VAR::<i32>)]
pub enum PlayerActionType {
   Unknown = -1,
   StartDestroyBlock = 0,
   AbortDestroyBlock = 1,
   ...
}
```